pkgs/by-name/te/tenv/package.nix

```
@@ -0,0 +1,26 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "tenv";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "tofuutils";
+    repo = "tenv";
+    rev = version;
```

```suggestion
    rev = "v${version}";
```
Fixes build of package.

---

Please, *always* re-build before pushing changes. Package was broken.

---

I executed:
```
nix-build $NIXPKGS -A tenv 
nix-shell -p nixpkgs-review --run "nixpkgs-review pr 297642"
```